src/main/java/de/rabea/ui/RealConsole.java

```diff
@@ -5,11 +5,11 @@
 import java.io.*;
 
 public class RealConsole implements Console {
-    BufferedReader userInput;
+    BufferedReader bufferedReader;
```

Comment:
You may want to make these instance variable private? Otherwise any object in the same package can see them and potentially change them.